src/Ledger/Certs/Properties.agda

```diff
@@ -131,3 +133,115 @@ instance
 
 Computational-CERTS : Computational _⊢_⇀⦇_,CERTS⦈_ String
 Computational-CERTS = it
+
+private variable
+  dCert : DCert
+  Γ : CertEnv
+  l : List DCert
+
+instance
+  _ = +-0-monoid
+
+  HasCoin-Map : ∀ {A} → ⦃ DecEq A ⦄ → HasCoin (A ⇀ Coin)
+  HasCoin-Map .getCoin s = ∑[ x ← s ] x
+
+≡ᵉ-getCoin : ∀ {A} → ⦃ _ : DecEq A ⦄ → (s s' : A ⇀ Coin) → s ˢ ≡ᵉ s' ˢ → getCoin s ≡ getCoin s'
+≡ᵉ-getCoin {A} ⦃ decEqA ⦄ s s' s≡s' = indexedSumᵛ'-cong {C = Coin} {x = s} {y = s'} s≡s'
```

Comment:
These are duplicated in `Ledger.Utxo`, either import them from there or move them into some file where both files import it from.

Reply:
Can I really import `HasCoin-Map` and `≡ᵉ-getCoin` from `Utxo`?  Doesn't the `Utxo` module require different parameters than `Certs.Properties`?  When I import them and try to use `HasCoin-Map`, Agda can't find the right instance... (sigh).

Reply:
Okay, thanks.  I have yet to fix the duplication of `HasCoin-Map` and `≡ᵉ-getCoin`, but will do so tomorrow (Thursday).  In the meantime, if you could look over the rest of the PR... I left some assumptions outstanding that I use in the proof of `CERTBASE-pov`.  Hopefully I can return to these and prove some of them soon, but I need to move on to pov for `LEDGERS` now.  (`CERTBASE-pov` took way too long and seemed much harder than necessary/expected. 😓)

Reply:
@WhatisRT I've rewritten the proofs.  I hope the statement and the proofs are correct now.  I also moved the `HasCoin-Map` and `≡ᵉ-getCoin` instances into `Ledger.Prelude`, so I think this PR is ready for a second look, when you have time.